web/src/pages/shoppingBasket.js

```diff
@@ -14,14 +14,15 @@ class ShoppingBasket extends Component {
         };
     }
 
-    addItem = price => {
+    addItem = (price, id) => {
         this.setState({
             numberOfSelectedItems: this.state.numberOfSelectedItems + 1,
             totalPrice: this.state.totalPrice + price,
         });
+        alert(this.state.selected);
```

Comment:
What is the purpose of this? Is this for debugging? Then, you should remove or make this comment after debugging. Additionally, there is no object named `selected` in `this.state` of this component.